cocos/core/math/quat.ts

```diff
@@ -411,6 +438,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en The quaternion calculated and the arc of rotation of the rotary shaft
```

Comment:
Calculates the quaternion from a given rotary shaft and a radian rotation around it.